spec/factories/state_file_nj_intakes.rb

```diff
@@ -105,5 +111,21 @@
     trait :df_data_many_w2s do
       raw_direct_file_data { StateFile::XmlReturnSampleService.new.read('nj_zeus_many_w2s') }
     end
+
+    trait :married do
+      filing_status { "married_filing_jointly" }
+      spouse_birth_date { Date.new(1990, 1, 1) }
+      spouse_ssn { "123456789" }
+    end
+
+    trait :primary_over_65 do
+      primary_birth_date { Date.new(1900, 1, 1) }
+    end
+
+    trait :married_spouse_over_65 do
```

Comment:
[dust] should this be `married_filing_jointly` or `mfj` over 65 instead so it's clearer, since there's also a married filing separately possibility?

Reply:
Great callout, thanks!